pmm/pmm2-server-release.groovy

```diff
@@ -177,17 +177,24 @@ pipeline {
                         fi
                         docker save percona/pmm-server:\${VERSION} | xz > pmm-server-\${VERSION}.docker
 
-                        #docker pull \${DOCKER_CLIENT_VERSION}
-                        #docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${VERSION}
-                        #docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:latest
-                        #docker push perconalab/pmm-client:\${VERSION}
-                        #docker save perconalab/pmm-client:\${VERSION} | xz > pmm-client-\${VERSION}.docker
+                        docker pull \${DOCKER_CLIENT_VERSION}
```

Comment:
We do not have DOCKER_CLIENT_VERSION option  for this job. Please add it.

Reply:
done